main.js

```diff
@@ -0,0 +1,73 @@
+let bookList = [
+  {
+    id: Date.now().toString(36) + Math.random().toString(36).substr(2, 3),
+    title: 'Black Clover',
+    author: 'Yuki Tabata',
+  },
+];
+
+if (localStorage.getItem('books')) {
+  bookList = JSON.parse(localStorage.getItem('books'));
+}
```

Comment:
- According to the project requirements, the bookList is used to originally save the list of books, I do not think there is any need to reassign the value of the bookList. Please fix this 👍 👍

Reply:
That part is to use the localStorage array, I don't think I should change it @mckent05